MemeMe/memeStruct.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  memeStruct.swift
```

Comment:
Good job moving the meme struct into another swift file. Files names should be upper camel case (MemeStruct.swift).